docs/source/api/v4/users.rst

```diff
@@ -90,6 +90,12 @@ Response Structure
 :stateOrProvince:   The name of the state or province where this user resides
 :tenant:            The name of the tenant to which this user belongs
 :tenantId:          The integral, unique identifier of the tenant to which this user belongs
+:ucdn:              The name of the :abbr:`uCDN (Upstream Content Delivery Network)` to which the user belongs
+
+	.. versionadded:: 6.2
```

Comment:
APIv4 is still in development, so you don't need to call out in which ATC version this field (or any field) was added. As far as any client is concerned, it's new and native to APIv4.0. Notice, for example, the type of the `role` field changed after 4.0 had been "released" as an unstable API, but it doesn't have a `.. versionchanged::` directive.